<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test</class>
 <widget class="QMainWindow" name="test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1422</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{background-color: rgb(255, 249, 243);
border-radius:20px;border:4px solid #b38144;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="settime">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 11pt &quot;Adobe 楷体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>931</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;幼圆&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>5,下列问题没有错误的一项是（ ）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>991</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="xu1">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Adobe 黑体 Std R&quot;;</string>
       </property>
       <property name="text">
        <string>A，第一段主要描写小石潭，开头先写竹，写小声，属于正面描写。</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="xu2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Adobe 黑体 Std R&quot;;</string>
       </property>
       <property name="text">
        <string>B、第二段写潭中鱼，正面写鱼侧面表现潭水的清流澈。</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="xu3">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Adobe 黑体 Std R&quot;;</string>
       </property>
       <property name="text">
        <string>C，第三段写小石潭的源流，采用比喻手法，形象地写出了溪岸的形状。</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="xu4">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Adobe 黑体 Std R&quot;;</string>
       </property>
       <property name="text">
        <string>D，第五段写同游的人，在写法上属于补叙。</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="nextT">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>680</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/bgrand/img/nextt.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backT">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>680</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/bgrand/img/upt.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>690</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/bgrand/img/bggrand.png);
font: 23pt &quot;幼圆&quot;;</string>
    </property>
    <property name="text">
     <string>交卷</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>60</y>
      <width>91</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>60</y>
      <width>91</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>60</y>
      <width>91</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4"/>
   </widget>
   <widget class="QPushButton" name="closed">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/bgrand/img/insertclose.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mins">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/bgrand/img/minsize.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
